build.sbt

```diff
@@ -38,8 +38,25 @@ lazy val scalaApiModels = project.in(file("api-models") / "scala")
       "com.gu" %% "content-api-models-scala" % contentApiModelsVersion
     ) ++ libraryDeps,
 
-    bintrayOrganization := Some("guardian"),
-    bintrayRepository := "mobile",
+    //publishTo := sonatypePublishToBundle.value
+    
+    // For snapshot
+    publishTo := sonatypePublishTo.value
```

Comment:
@alexduf  Is this going to publish to snapshot repository?

Reply:
That's a really good question, I'm afraid I don't know the answer to that!